fearless/Common/Services/EraCountdownService/EraCountdownService.swift

```diff
@@ -0,0 +1,203 @@
+import RobinHood
+import FearlessUtils
+import SoraKeystore
+
+protocol EraCountdownServiceProtocol {
+    func fetchCountdownOperationWrapper() -> CompoundOperationWrapper<UInt64>
+}
+
+enum EraCountdownServiceError: Error {
+    case noData
+}
+
+final class EraCountdownService: EraCountdownServiceProtocol {
+    let chain: Chain
+    let runtimeCodingService: RuntimeCodingServiceProtocol
+    let storageRequestFactory: StorageRequestFactoryProtocol
+    let engine: JSONRPCEngine
+
+    init(
+        chain: Chain,
+        runtimeCodingService: RuntimeCodingServiceProtocol,
+        storageRequestFactory: StorageRequestFactoryProtocol,
+        engine: JSONRPCEngine
+    ) {
+        self.chain = chain
+        self.runtimeCodingService = runtimeCodingService
+        self.storageRequestFactory = storageRequestFactory
+        self.engine = engine
+    }
+
+    func fetchCountdownOperationWrapper() -> CompoundOperationWrapper<UInt64> {
```

Comment:
In some cases, for example, for unbondings and payouts we need to display time from now till some era. In current solution we need first to fetch remained time from in current era and then calculate difference between eras. And so there are two problems: 1) we should do several steps 2) we use hardcoded assumption about era duration. My suggestion is to include here an optional closure parameter (so we could chain with other operations) that returns destination era.